lib/plugins/filter/post_permalink.js

```diff
@@ -13,6 +13,7 @@ function postPermalinkFilter(data){
     id: data.id || data._id,
     title: data.slug,
```

Comment:
I think that the proper fix would be to change this line, to this:

    title: util.slugize(data.title, {transform: 1}),

... but that could break existing setups :disappointed: